server/src/main/java/org/opensearch/common/geo/GeoPoint.java

```diff
@@ -112,18 +111,16 @@ public GeoPoint resetFromString(String value, final boolean ignoreZValue, Effect
         return parseGeoHash(value, effectivePoint);
     }
 
-
     public GeoPoint resetFromCoordinates(String value, final boolean ignoreZValue) {
         String[] vals = value.split(",");
         if (vals.length > 3) {
-            throw new OpenSearchParseException("failed to parse [{}], expected 2 or 3 coordinates "
-                + "but found: [{}]", vals.length);
+            throw new OpenSearchParseException("failed to parse [{}], expected 2 or 3 coordinates " + "but found: [{}]", vals.length);
         }
         final double lat;
         final double lon;
         try {
             lat = Double.parseDouble(vals[0].trim());
-         } catch (NumberFormatException ex) {
+        } catch (NumberFormatException ex) {
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

**Problem**: While wrapping the caught exception into a custom one, information about the caught exception is being lost, including information about the stack trace of the exception.

**Fix**: If the caught exception object does not contain sensitive information, consider passing it as the &#34;rootCause&#34; or inner exception parameter to the constructor of the new exception before throwing the new exception. (Note that not all exception constructors support inner exceptions. Use a wrapper exception that supports inner exceptions.) 
[Learn more](https://www.ibm.com/support/pages/best-practice-catching-and-re-throwing-java-exceptions)
Similar issue at line numbers 128.